edc-tests/edc-controlplane/agreement-retirement-tests/src/test/java/org/eclipse/tractusx/edc/tests/agreement/retirement/RetireAgreementTest.java

```diff
@@ -131,6 +131,12 @@ void retireAgreement_shouldCloseTransferProcesses() {
 
         }
 
+        @Test
+        @DisplayName("Verify non existing Contract Agreement is not stored in retired table.")
```

Comment:
please remove this @DisplayName as it's not providing any additional interesting information to the test name

Reply:
was already removed in previous commit as indicated [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1631#discussion_r1810878681).